### PR TITLE
docs(fx113-relnote): add "prefers-reduced-transparency" to experimental feature list

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

Fx113 supports `prefers-reduced-transparency` media query behind the preference `layout.css.prefers-reduced-transparency.enabled`.

This PR:
- adds an entry for `prefers-reduced-transparency` media query to the Firefox experimental feature list for release notes.
- adds `prefers-reduced-transparency` to the [Using media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) page.

### Related issues and pull requests

- Doc issues: https://github.com/mdn/content/issues/25773 and https://github.com/mdn/content/issues/26150
- Content update: https://github.com/mdn/content/pull/25289

### Related links

- Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1736914
- Spec: https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-transparency

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -759,6 +759,47 @@ See ([Firefox bug 1673987](https://bugzil.la/1673987)) for more details.
   </tbody>
 </table>
 
+### prefers-reduced-transparency media feature
+
+The CSS [`prefers-reduced-transparency`](/en-US/docs/Web/CSS/@media/prefers-reduced-transparency) media feature lets you detect if a user has enabled the setting to reduce transparency on their device.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:764]: Minor one:
```suggestion
The CSS [`prefers-reduced-transparency`](/en-US/docs/Web/CSS/@media/prefers-reduced-transparency) media feature lets you detect if a user has enabled the setting to minimize the amount of transparent or translucent layer effects on their device.
```

**Review by bsmth — APPROVED** [review]
One comment outstanding, looks good otherwise!

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/mozilla/firefox/experimental_features/index.md:764]: Looks good 👍

**Comment by dipikabh** [comment]
Thanks for the review @bsmth!

**Pull request merged**